requirements.txt

```diff
@@ -1,2 +1,3 @@
 karton-core>=4.2.0,<5.0.0
 SFlock2==0.3.16
+six
```

Comment:
should we pin it to a specific version?

Reply:
Previously it was unpinned, so I just recovered it to the original version (see the PR full diff).
I don't think it's strongly required to pin it, it's just Py2/3 compatibility library and I don't expect any breaking changes there.